wiki/Contests/FBC/en.md

```diff
@@ -0,0 +1,158 @@
+---
+tags:
+- FBC
+- Mapping 
+---
+
+# French Beatmapping Contest
+
+The **French Beatmapping Contest** (***FBC***) is a mapping contest for French-speaking mappers. In each edition, mappers have to create a difficulty of Insane or higher.
+
+Points are attributed following these judging criteria: **Structure**, **Relevance**, **Creativity**, **Judge Impression** and **Hitsounds**. These criteria may change if necessary. 
+
+The top three mappers of a tournament will receive an osu!supporter tag, and the winner of each edition will earn a personalized badge that will be shown on their osu! profile.  
+
+Since the 2017 edition, the chosen songs have a length lower than 3 mintutes and the difficulties under the Insane level are mapped by the tournament hosts or by members from the French mapping community.
```

Comment:
```suggestion
Since the 2017 edition, the chosen songs have a length lower than 3 minutes and the difficulties under the Insane level are mapped by the tournament hosts or by members from the French mapping community.
```